### ISSUE TITLE
Color theme

### ISSUE THREAD

**Issue by allenwoods** [yscoder/hexo-theme-indigo#31]
Can Color theme be changed from indigo to other, such as blue or teal ?

**Comment by yscoder** [comment]
Yes, you can!  Change the file's color variable,  `themes\indigo\source\css\_partial\variable.less`.

**Comment by allenwoods** [comment]
Thanks for the great theme！It will be better if alternative color themes provided

**Comment by yscoder** [comment]
[Material Design Color Palette Generator](http://www.materialpalette.com/)
This site can help you configure favorite color theme.
